src/components/mark/mark.tsx

```diff
@@ -8,11 +8,17 @@
 
 import React, { HTMLAttributes, FunctionComponent, ReactNode } from 'react';
 import classNames from 'classnames';
+import { useEuiI18n } from '../i18n';
 import { CommonProps } from '../common';
 import { useEuiTheme } from '../../services';
 import { euiMarkStyles } from './mark.styles';
 export type EuiMarkProps = HTMLAttributes<HTMLElement> &
   CommonProps & {
+    /**
+     * Set to `false` to remove the CSS :before and :after
+     * screen reader helper text
+     */
+    hasHighlightText?: boolean;
```

Comment:
Nit picky naming, but I'd rather have this be something named more screen-reader specific since I don't know what "highlight" here means in context of `EuiMark`
```suggestion
    hasScreenReaderHelpText?: boolean;
```

or something shorter... naming. hard.